policyengine_us/parameters/gov/local/ny/nyc/tax/income/credits/cdcc/rate.yaml

```diff
@@ -0,0 +1,118 @@
+description: NYC CDCC Limitation Table Rate
+metadata:
+  type: single_amount
+  threshold_unit: currency-USD
+  amount_unit: /1
+  label: NYC CDCC
+  reference:
+    - title: NY State/City CDCC Form 216 Instructions
+      href: https://www.tax.ny.gov/pdf/current_forms/it/it216i.pdf#page=7
```

Comment:
Have you searched for a NYC legal code reference? They might define this structure more succinctly in the law.

Reply:
I was having trouble finding it in the legal code, but we could write it as a formula if that would be better? The first and last steps are .015, but otherwise it decreases by .03 for every $200 in additional AGI. In this case it'd be a variable instead of a parameter?

Reply:
Looks like it's in [§ 11-1706 (e)](https://codelibrary.amlegal.com/codes/newyorkcity/latest/NYCadmin/0-0-0-13608)